go.mod

```diff
@@ -0,0 +1,3 @@
+module example.com/m/v2
```

Comment:
We should add github path of this package in front of module:-

Reference:- https://go.dev/doc/tutorial/getting-started#code